core/common/modules/connect/apps/base-app.php

```diff
@@ -184,7 +184,7 @@ public function action_get_token() {
 			$this->redirect_to_admin_page();
 		}
 
-		//phpcs:ignore WordPress.Security.NonceVerification.Recommended -- Nonce is verified in line 200.
+		//phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
Where is the nonce verified?

Reply:
The comment was already there. 
`//phpcs:ignore WordPress.Security.NonceVerification.Recommended`

Should we check for nonce?

Reply:
Please check where is the verification and explain in the comment, here and for each nonce ignoring

Reply:
Done